hazelcast/src/main/java/com/hazelcast/config/MemoryTierConfig.java

```diff
@@ -94,9 +94,17 @@ public String toString() {
                 + '}';
     }
 
+    public long getCapacityValue() {
+        return capacity.getValue();
+    }
+
+    public int getCapacityMemoryUnitType() {
+        return capacity.getUnit().ordinal();
+    }
+
```

Comment:
Maybe instead of these, we can add a new custom type to protocol definition for `Capacity` and change the `MemoryTierConfig` so that it has a single parameter of type `Capacity`. WDYT?